joomla/joomla-cms
# [5.0] schema.org cleanup

### Summary of Changes
- Moved cleanup functions to individual traits
- Added traits to each plugin
- Improved blog/categories views
- Added global cleanup to schemaorg system plugin


### Testing Instructions
- Create different articles with schema information
- Check head area of your page


### Actual result BEFORE applying this Pull Request
- empty values
- wrong format for date/image/durations


### Expected result AFTER applying this Pull Request
- No empty values
- Correct image/date/duration compilation


### Link to documentations
Please select:
